data_steward/analytics/cdr_ops/vocabulary_upload_qc.py

```diff
@@ -98,27 +107,38 @@
                    vocabulary_dataset_new=vocabulary_dataset_new)
 execute(client, query, max_rows=True)
 
+# # Verify the updates made to the vocabulary in the upload process
+
+# ## All AoU_Custom and AoU_General concepts are present in the new vocabulary
+# In the process of updating the vocabulary custom concepts are added to the vocabulary. <br>
+# The list of concepts should have been updated at the start of this notebook.<br>
+#
+# If the check fails, investigate. It is important that all of the custom concepts are added to the vocabulary.
+
 # +
-# How has relationship changed?
-
-# changed_ppi_relationships=execute(f'''
-# SELECT SUM(CASE WHEN r1.is_hierarchical  != r2.is_hierarchical THEN 1 ELSE 0 END) change_hierarchy,
-#        SUM(CASE WHEN r1.defines_ancestry != r2.defines_ancestry THEN 1 ELSE 0 END) change_ancestry,
-#        SUM(CASE WHEN r1.reverse_relationship_id != r2.reverse_relationship_id THEN 1 ELSE 0 END) change_reverse_rel,
-#        SUM(CASE WHEN r1.relationship_concept_id != r2.relationship_concept_id THEN 1 ELSE 0 END) change_rel_concept
-# FROM `{vocabulary_dataset_old}.relationship` r1
-# JOIN `{vocabulary_dataset_new}.relationship` r2 ON r1.relationship_id = r2.relationship_id
-# JOIN `{vocabulary_dataset_new}.concept` c ON r1.relationship_concept_id = c.concept_id
-# WHERE c.vocabulary_id LIKE "PPI"
-# ''')
+tpl = JINJA_ENV.from_string('''
+SELECT
+COUNT(concept_code) as total
+FROM `{{vocabulary_dataset_new}}.concept` c
+WHERE vocabulary_id in ('AoU_Custom', 'AoU_General')
+OR concept_code IN ('AOU generated') 
+''')
+query = tpl.render(vocabulary_dataset_old=vocabulary_dataset_old,
+                   vocabulary_dataset_new=vocabulary_dataset_new,
+                  custom_concepts=custom_concepts)
+df = execute(client, query, max_rows=True)
+
+display(df)
+print('This check passes if the total is ' + str(len(custom_concepts)))
```

Comment:
This statement may be hard to miss. Maybe an IF ELSE with COUNT(*)>{str(len(custom_concepts))}?

Reply:
I changed a few things to make this easier to see the pass/fail status and troubleshooting. Let me know if you have any more suggestions on these changes. Thanks!